src/app/tab/loadshedding/loadshedding.component.html

```diff
@@ -1,16 +1,19 @@
-Test
-<div *ngFor="let area of cachedAreaInfo">
-  <h2>{{area.info.name}}</h2>
-  <p>{{area.info.region}}</p>
-  <p>{{area.schedule.source}}</p>
-  <h3>Currently: {{area.events[0].note}}</h3>
-  <p>{{area.events[0].start}} - {{area.events[0].end}}</p>
-  <ng-container *ngFor="let day of area.schedule.days">
-    <h3>{{day.name}} ({{day.date}})</h3>
-    <ul>
-      <li *ngFor="let stage of day.stages; let i = index">
-        Stage {{i + 1}}: <ng-container *ngFor="let time of stage">{{time}}, </ng-container>
-      </li>
-    </ul>
-  </ng-container>
-</div>
+<div class='loadsheddingContainer'>
+   <div class="suburb">
+    <div class="header">
+      <h2>{{cachedAreaInfo?.data?.info?.name}} </h2>
+      <label class="dim">{{cachedAreaInfo?.data?.info?.region}}</label>
+    </div>
+    <div class="schedules">
+      <ng-container *ngFor="let day of cachedAreaInfo.data?.schedule?.days">
+        <div class="daySchedule" *ngIf="showData(day).length >0">
```

Comment:
Getting an error when loading a tab that I'm pretty sure is related to this line: 

```
main.js:9070 ERROR TypeError: Cannot read properties of undefined (reading 'length')
    at Object.updateDirectives (main.js:9070:817)
```

Reply:
I found another bug also. Would have to spend some time sorting it out.

Reply:
Sorted.

Reply:
There was still a bug here, looks like it just needs this ngIf to be changed to `*ngIf="showData(day) && showData(day).length >0"`

I've got some time today to work on this though so I'll take what you have and make that fix and a couple small styling things and get this going. Thank you so much for the work on this!